### DISCUSSION TITLE
怎么加载模型报错捏

### DISCUSSION THREAD

**Discussion by HuangHua123** [zai-org/ChatGLM3#1060] [Q&A] [answered]
Unable to load weights from pytorch checkpoint file for '/mnt/pytorch_model-00001-of-00007.bin' at '/mnt/pytorch_model-00001-of-00007.bin'. If you tried to load a PyTorch model from a TF 2.0 checkpoint, please set from_tf=True.

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
检查一下模型是否正常下载，大概率是模型没下全
